README.md

```diff
@@ -21,7 +21,97 @@ Use robust linting like we do
 
 ## Contributing
 
-It uses [lerna](https://lernajs.io) to maintain all packages within one repository, so take a look at its features. 
+It uses [lerna](https://lernajs.io) to maintain all packages within one repository, so take a look at its features.
+
+## How to install
+
+### Linter for React projects `eslint-config-datarockets`
+
+[![npm version](https://img.shields.io/npm/v/eslint-config-datarockets.svg?style=flat-square)](https://www.npmjs.com/package/eslint-config-datarockets)
```

Comment:
Hm... Seems like we have a single version for all the packages: https://github.com/datarockets/style-guide/pull/6/files#diff-04c6e90faac2675aa89e2176d2eec7d8R116. So we can remove those badges in each section.

Reply:
👍